oracle_cardano/processor/tx_processors/success/bridging_requested_processor_skyline.go

```diff
@@ -0,0 +1,353 @@
+		} else {
+			receiverAmount = receiver.Additional.DestAmount
+			receiverAmountWrapped = receiver.Amount
+
+			totalAmountSrc += receiver.Amount + receiver.Additional.SrcAmount
+			totalAmountWrappedTokenSrc += uint64(0)
+
+			totalAmountDest += receiver.Additional.DestAmount
+			totalAmountWrappedTokenDest += receiver.Amount
+		}
+
+		receivers = append(receivers, cCore.BridgingRequestReceiver{
+			DestinationAddress: receiverAddr,
+			Amount:             new(big.Int).SetUint64(receiverAmount),
+			AmountWrapped:      new(big.Int).SetUint64(receiverAmountWrapped),
+		})
+	}
+
+	totalAmountDest += metadata.FeeAmount.DestAmount
+	totalAmountSrc += metadata.FeeAmount.SrcAmount
+
+	receivers = append(receivers, cCore.BridgingRequestReceiver{
+		DestinationAddress: feeAddress,
+		Amount:             new(big.Int).SetUint64(metadata.FeeAmount.DestAmount),
+		AmountWrapped:      new(big.Int).SetUint64(0),
+	})
+
+	claim := cCore.BridgingRequestClaim{
+		ObservedTransactionHash:         tx.Hash,
+		SourceChainId:                   common.ToNumChainID(tx.OriginChainID),
+		DestinationChainId:              common.ToNumChainID(metadata.DestinationChainID),
+		Receivers:                       receivers,
+		NativeCurrencyAmountDestination: new(big.Int).SetUint64(totalAmountDest),
+		NativeCurrencyAmountSource:      new(big.Int).SetUint64(totalAmountSrc),
+		WrappedTokenAmountSource:        new(big.Int).SetUint64(totalAmountWrappedTokenSrc),
+		WrappedTokenAmountDestination:   new(big.Int).SetUint64(totalAmountWrappedTokenDest),
+		RetryCounter:                    big.NewInt(int64(tx.BatchFailedCount)),
+	}
+
+	claims.BridgingRequestClaims = append(claims.BridgingRequestClaims, claim)
+
+	p.logger.Info("Added BridgingRequestClaim",
+		"txHash", tx.Hash, "metadata", metadata, "claim", cCore.BridgingRequestClaimString(claim))
+}
+
+/*
+func (*BridgingRequestedProcessorSkylineImpl) addRefundRequestClaim(
+	claims *core.BridgeClaims, tx *core.CardanoTx, metadata *common.BridgingRequestMetadata,
+) {
+
+		var outputUtxos []core.Utxo
+		for _, output := range tx.Outputs {
+			outputUtxos = append(outputUtxos, core.Utxo{
+				Address: output.Address,
+				Amount:  output.Amount,
+			})
+		}
+
+		// what goes into UtxoTransaction
+		claim := core.RefundRequestClaim{
+			TxHash:             tx.Hash,
+			RetryCounter:       0,
+			RefundToAddress:    metadata.SenderAddr,
+			DestinationChainId: metadata.DestinationChainId,
+			OutputUtxos:        outputUtxos,
+			UtxoTransaction:    core.UtxoTransaction{},
+		}
+
+		claims.RefundRequest = append(claims.RefundRequest, claim)
+
+		p.logger.Info("Added RefundRequestClaim",
+		"txHash", tx.Hash, "metadata", metadata, "claim", core.RefundRequestClaimString(claim))
+}
+*/
+
+func (p *BridgingRequestedProcessorSkylineImpl) validate(
+	tx *core.CardanoTx, metadata *common.BridgingRequestMetadata, appConfig *cCore.AppConfig,
+) error {
+	multisigUtxo, err := utils.ValidateTxOutputs(tx, appConfig, false)
```

Comment:
I think you may need to first validate:
        cardanoSrcConfig, _ := cUtils.GetChainConfig(appConfig, tx.OriginChainID)
	if cardanoSrcConfig == nil {
		return fmt.Errorf("unsupported chain id found in tx. chain id: %v", tx.OriginChainID)
	}

and then: 
        multisigUtxo, err := utils.ValidateTxOutputs(tx, appConfig, false)
	if err != nil {
		return err
	}

Reply:
I was just fixing this. Do you think its better to do it this way, or to check if chain config is nil in utils.ValidateTxOutputs, since we are using it in multiple places, and it can be easy to forget to check that before calling the method?

Reply:
keep it like this for now, because it is used on reactor also. we dont want to diverge now